tools/nnabla

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -e
+pushd $(cd $(dirname ${0})/..; pwd) > /dev/null
+
+case "${OSTYPE}" in
+    msys*) python="winpty python";;
+    *) python="python";;
+esac
+
+venv() {
+    env_dir=./third_party/env/nnabla
+    [ -d "${env_dir}" ] || ${python} -m venv ${env_dir}
+    case "${OSTYPE}" in
+        msys*) source ${env_dir}/Scripts/activate;;
+        *) source ${env_dir}/bin/activate;;
+    esac
+    ${python} -m pip install --quiet --upgrade pip
+}
+
+clean() {
+    echo "nnabla clean"
+    rm -rf "./third_party/source/nnabla"
+}
+
+sync() {
+    echo "nnabla sync"
+    mkdir -p "./third_party/source/nnabla/build-tools/code_generator"
+    curl --silent --location --output "./third_party/source/nnabla/build-tools/code_generator/functions.yaml" "https://github.com/sony/nnabla/blob/master/build-tools/code_generator/functions.yaml?raw=true"
+    curl --silent --location --output "./third_party/source/nnabla/build-tools/code_generator/solvers.yaml" "https://github.com/sony/nnabla/blob/master/build-tools/code_generator/solvers.yaml?raw=true"
+    mkdir -p "./third_party/source/nnabla/src/nbla/proto"
+    curl --silent --location --output "./third_party/source/nnabla/src/nbla/proto/nnabla.proto.tmpl" "https://github.com/sony/nnabla/blob/master/src/nbla/proto/nnabla.proto.tmpl?raw=true"
+    awk -i inplace '/syntax = "proto3";/ { print; print "package nnabla;"; next }1' ./third_party/source/nnabla/src/nbla/proto/nnabla.proto.tmpl
+}
+
+install() {
+    echo "nnabla install"
+    venv
+    ${python} -m pip install --quiet --upgrade six mako pyyaml
+    deactivate
+}
+
+schema() {
+    echo "nnabla schema"
+    venv
+    export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+    ${python} ./tools/nnabla-script.py proto
```

Comment:
There shouldn't be a requirement on Python or `mako` to generate the schema file. Netron supports dozens of formats and shouldn't be blocked on complex build steps. Ideally the latest expanded version of the `.proto` file should be provided by the nnabla repo.